packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -150,3 +152,29 @@ export class CSetLoanToken extends ComposableBuffer<SetLoanToken> {
     ]
   }
 }
+
+/**
+ * CreateVault DeFi Transaction
+ */
+export interface CreateVault {
+  ownerAddress: Script // --------------------| n = VarUInt{1-9 bytes}, + n bytes, Vault's owner address
+  schemeId: string // ------------------------| c = VarUInt{1-9 bytes}, + c bytes UTF encoded string, Vault's loan scheme id
+  isUnderLiquidation: boolean // -------------| 1 byte, Vault is under liquidation
```

Comment:
```suggestion
  isUnderLiquidation: false // -------------| 1 byte, Vault is under liquidation
```
field is mandatory when decoding from existed block, but new creation should not allow it.

This change will basically impact only test case since the type check is not on runtime.
Then client (eg light wallet) can extend client side API like `createVault(Omit<CreateVault, 'isUnderLiquidation'>)`

Reply:
once Antoniy adds the validation on the ain side, we can rely on that. will wait until he makes a PR

Reply:
https://github.com/DeFiCh/ain/pull/740

Reply:
might require more changes from Antoniy's PR